logging.go

```diff
@@ -25,9 +26,14 @@ func prefixWithReplicationId(lr LoggingReplication, raw string) string {
 	replicationId := lr.Parameters.ReplicationId
 
 	if replicationId != "" {
-		return strings.Join([]string{"[",replicationId ,"] ", raw}, "")
+		return strings.Join([]string{"[", replicationId, "] ", raw}, "")
```

Comment:
Yep, agreed.  Also in a separate PR, we should run every file in this repo through `go fmt`